presto-main/src/main/java/com/facebook/presto/execution/StageStateMachine.java

```diff
@@ -236,6 +237,7 @@ public StageInfo getStageInfo(Supplier<Iterable<TaskInfo>> taskInfosSupplier, Su
 
             cumulativeMemory += taskStats.getCumulativeMemory();
             totalMemoryReservation += taskStats.getMemoryReservation().toBytes();
+            totalRevocableMemoryReservation += taskStats.getRevocableMemoryReservation().toBytes();
```

Comment:
it's consistent with other places in code (like `StateStats`, `DriverStats`, etc) and with other stats. There is for example `totalCpuTime`, `totalBlockedTime` and `totalUserTime`. `total` means "sum of all values from child entities" (driver sums it's operators values, stage sums it's tasks etc...)